strum_macros/src/macros/enum_discriminants.rs

```diff
@@ -130,7 +126,7 @@ pub fn enum_discriminants_inner(ast: &DeriveInput) -> syn::Result<TokenStream> {
     Ok(quote! {
         /// Auto-generated discriminant enum variants
         #derives
-        #(#pass_though_attributes)*
+        #(#[ #pass_though_attributes ])*
```

Comment:
Could you help me understand the difference here?

Reply:
`#()` interpolations can contain arbitrary extra tokens in addition to an iterable of `impl ToTokens`. So instead of 

```rust
let values = vec![1, 2, 3];
let push_values: Vec<_> = values.into_iter().map(|v| quote! { vec.push(#v); }).collect();
quote! {
    let vec = Vec::new();
    #(#push_values)
}
```

you can just write

```rust
let values = vec![1, 2, 3];
quote! {
    let vec = Vec::new();
    #( vec.push(#values); )
}
```

I did the same kind of simplification here ([context](https://github.com/Peternator7/strum/pull/134/files/15596f68516f2e6728fb7cd59507f3bb63482d21#diff-189e994b49682e22ee918c0157e1638eR40-R41)).